src/dispatcher/mod.rs

```diff
@@ -159,6 +167,20 @@ impl Dispatcher {
                     name
                 )));
             }
+            // check if service is not yet up but started
+            if let Some(_) = service.url {
+                return Err(ServicingError::ClusterProvisionError(format!(
+                    "Service {} is starting",
+                    name
+                )));
+            }
+            match force {
```

Comment:
See above. You can remove this by using force directly.

Reply:
not sure what you mean .-.

Reply:
by using force i can skip validation flow but still need to skip prompt